src/pages/Beta/Stake/PoolCard/StakeWidget/index.tsx

```diff
@@ -67,14 +66,15 @@ const StakeWidget: React.FC<Props> = ({ stakingInfo, onClose }) => {
               />
 
               <Box ml="5px" mt="25px">
-                <Percentage
-                  onChangePercentage={value => {
+                <NumberOptions
+                  onChange={value => {
                     setStepIndex(value)
-                    onChangePercentage(value * 25)
+                    onChangePercentage(value)
                   }}
                   currentValue={stepIndex}
```

Comment:
can you please make sure by default we select 100% as per my previous comment ? https://github.com/pangolindex/interface/pull/447/files#r855729532